### PR TITLE
Settlement report adjustment disable add remove

### PR DESCRIPTION
**Pull request by msbodetti**
### References Issues:
https://app.asana.com/0/1158142652422330/1164097677491900/f
### Description:
The adjustment add and delete buttons didn't check the status of the settlement report i.e. Pending or Finalized.

Pending: Can add/remove adjustments
Finalized: Cannot add/remove adjustments
### Release Screenshots / Video:
https://timo614adjustsettlementfinalizationtousejob.scratch.bigneon.com/admin/reports/settlement-list
### Environment Variables:
 * No change

### CODE REVIEW

File: src/components/pages/admin/reports/settlement/AdjustmentsList.js [hunk 1]
```diff
@@ -93,13 +93,15 @@ class AdjustmentsList extends Component {
...
-								</IconButton>
+								{status === "Pending" ? (
+									<IconButton
+										className={classes.icon}
+										onClick={this.onDeleteDialogOpen.bind(this, id)}
+										iconUrl="/icons/delete-gray.svg"
+									>
+										Delete
+									</IconButton>
+								) : null}
```
Comment (Krakaw) [inline src/components/pages/admin/reports/settlement/AdjustmentsList.js]:
```suggestion
								{status === "Pending" && (
									<IconButton
										className={classes.icon}
										onClick={this.onDeleteDialogOpen.bind(this, id)}
										iconUrl="/icons/delete-gray.svg"
									>
										Delete
									</IconButton>
								)}
```

**Comment by msbodetti** [comment]
Thanks @Krakaw

File: src/components/pages/admin/reports/settlement/AdjustmentsList.js [hunk 2]
```diff
@@ -93,13 +93,15 @@ class AdjustmentsList extends Component {
...
 								</span>{" "}
 								- {displayCreatedAt}
-								<IconButton
-									className={classes.icon}
-									onClick={this.onDeleteDialogOpen.bind(this, id)}
-									iconUrl="/icons/delete-gray.svg"
-								>
-									Delete
-								</IconButton>
+								{status === "Pending" && (
```
Comment (Krakaw) [inline src/components/pages/admin/reports/settlement/AdjustmentsList.js]: Use the `bn-api-node` `SETTLEMENT_STATUS` enums. http://github.com/big-neon/bn-api-node/blob/e6438f246ae951089adc7a6b5a0865231da86443/src/interfaces/enums/settlements.enum.ts#L2-L2

File: src/components/pages/admin/reports/settlement/GrandTotalsTable.js [hunk 3]
```diff
@@ -61,7 +62,7 @@ const GrandTotalsTable = props => {
 				{[
 					<span key={"adjustments"} className={classes.adjustmentText}>
 						Adjustments&nbsp;
-						{onAddAdjustment ? (
+						{onAddAdjustment && status === "Pending" ? (
```
Comment (Krakaw) [inline src/components/pages/admin/reports/settlement/GrandTotalsTable.js]: Use bn-api-node enum

**Pull request merged**